Noemi_first_try/index/style.css

```diff
@@ -0,0 +1,121 @@
+    body {
+      margin: 0;
+    }
+    header {
+      background-image: url("./mann_telefon.jpg");
+      background-size: cover;
+      height: 110px;
+      padding-top: 230px;
+      padding-left: 500px;
+      margin-bottom: 10px;
+    }
+    p{
+      font-family: 'Times New Roman', Times, serif;
+      text-align: justify;
+      width: 400px;
+      margin: 20px auto 20px auto;
+      font-weight: normal;
+    }
+    home{
+    color: rgb(243, 156, 24);
+    font-size: 20px;
+    padding-left: 1200px;
+    }
+    h1 {
+      margin: 0;
+      color:rgb(255,207,0);
+      font-size: 50px;
+      font-family: Georgia, 'Times New Roman', Times, serif;
+    }
+    h2 {
+    color: black;
+    font-size: 50px;
+    font-family: Georgia, 'Times New Roman', Times, serif;
+    padding-left: 30px;
+    }
+    h3{
+      color: black;
+      font-size: 20px;
+      font-family: Georgia, 'Times New Roman', Times, serif;
+      text-align: right;
+      padding-top: 150px;
+    }
+    .box1{
+      float: left;
+      width: 21%;
+      height: 240px;
+      padding: 20px;
+      margin: 25px;
+      background: rgb(145,48,75);
+      box-sizing: border-box;
+    }
+    .box2{
+      float: left;
+      width: 21%;
+      height: 240px;
+      padding: 20px;
+      margin: 25px;
+      background:rgb(188,34,81);
+      box-sizing: border-box; 
+    }
```

Comment:
The styling of the boxes with the different colors looks nice! 

For boxes 1 to 8 I would suggest shortening the code (using the DRY principle), for example like:
`.box-one, .box-two, .box-three...box-eight {`
`    float: left;`
`      width: 21%;`
`      height: 240px;`
`      padding: 20px;`
`      margin: 25px;`
`     box-sizing: border-box;`
`}`
Then you would only need to change the background (color) for each box, for example:
`.box-one {`
`background: rgb(188,34,81);`
`} `